browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -1140,8 +1141,9 @@ void VerticalTabStripRegionView::OnResize(int resize_amount,
       (*vertical_tab_on_right_ ? bounds_in_screen.right() - cursor_position
                                : cursor_position - bounds_in_screen.x()) -
       *resize_offset_ - GetInsets().width();
-  dest_width = std::clamp(dest_width, tab_style_->GetPinnedWidth() * 3,
-                          tab_style_->GetStandardWidth() * 2);
+  bool is_split = tabs::features::HorizontalTabsUpdateEnabled();
```

Comment:
checking this change here and other places in this commit.
`is_split` and horizontal tab enabled state is not related condition.